hedera-node/hedera-evm/src/test/java/com/hedera/node/app/service/evm/contracts/operations/HederaEvmCreate2OperationTest.java

```diff
@@ -88,7 +88,7 @@ void enabledOnlyIfCreate2IsEnabled() {
     void computesExpectedTargetAddress() {
         final var expectedAddress = Address.BLS12_G1ADD;
         final var canonicalSource = Address.BLS12_G1MULTIEXP;
-        final var besuOp = new Create2Operation(gasCalculator);
+        final var besuOp = new Create2Operation(gasCalculator, 48 * 1024);
```

Comment:
Q: Is this 48kb? WHat's the significance of the choice?

Reply:
(a) the API changed, so we needed some value
(b) 48k is defined in EIP-3860
(c) it's test code, this isn't used but we need some value for the constructor.